src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -1324,6 +1476,34 @@ int RadosBucket::commit_logging_object(const std::string& obj_name,
       }
       return ret;
     }
+  } else if (is_ec_pool){
+    ret = rgw_put_system_obj(dpp, store->svc()->sysobj,
+                             data_pool,
+                             temp_obj_name,
+                             bl_data,
+                             true,
+                             nullptr,
+                             mtime,
+                             y,
+                             &obj_attrs);
+
+    if (ret < 0) {
```

Comment:
if this fails with `-EEXISTS` we probably want to continue with the deletio nand commit.
could be that someone did the copy and then failed?
o/w we will get stuck here